ydb/docs/en/core/dev/troubleshooting/schemas/overloaded-shards.md

```diff
@@ -0,0 +1,23 @@
+# Overloaded shards
+
+## Description
+
+[Data shards](../../../concepts/glossary.md#data-shard) in {{ ydb-short-name }} might get overloaded for the following reasons:
+
+* A table is created without the [AUTO_PARTITIONING_BY_LOAD](../../../concepts/datamodel/table.md#AUTO_PARTITIONING_BY_LOAD) clause.
+
+    In this case {{ ydb-short-name }} does not split overloaded shards. If a shard has to process too many queries, it will use up all its CPU resources and, consequently, query response times will increase.
+
+* A table is created with the [AUTO_PARTITIONING_MAX_PARTITIONS_COUNT](../../../concepts/datamodel/table.md#AUTO_PARTITIONING_MAX_PARTITIONS_COUNT) clause and has already reached the limit.
```

Comment:
```suggestion
* A table was created with the [AUTO_PARTITIONING_MAX_PARTITIONS_COUNT](../../../concepts/datamodel/table.md#AUTO_PARTITIONING_MAX_PARTITIONS_COUNT) setting and has already reached the partition limit.
```

Reply:
In SQL standard, queries consist of *statements* separated by semicolons (like `SELECT` statement, `INSERT INTO` statement, etc.), and *clauses* are syntactic parts of statements (`GROUP BY` clause, `ORDER BY` clause, etc.). So, in this case, there would be a `WITH` clause, and these `AUTO_PARTITIONING_MAX_PARTITIONS_COUNT` are more like settings for it.